_posts/2020-04-08-release-2.3.0.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: default
+title: Release v2.3.0
+tags: ["v2.3.0", "changelog"]
+excerpt_separator: <!--more-->
+---
+
+As of 08/04 2020 the new version 2.3.0 is out
+<!--more-->
+
+## Standardization & Conformance Improvements
+
+- [#1222]({{ site.github.issues_url }}/1222) Spark version checking for Conformance and Standardization (currently supported versions >=2.4.2, <3.0.0)
+- [#1157]({{ site.github.issues_url }}/1260) The group explosion strategy for mapping conformance rules is turned on by default. 
```

Comment:
these numbers don't align. 1157 vs 1260

Reply:
Thanks, nice catch. (I fixed a linked PR instead of issue, but not fully.)